src/utils/uint256.ts

```diff
@@ -26,6 +42,14 @@ export function isUint256(bn: BigNumberish): boolean {
 /**
  * Convert BigNumberish (string | number | bigint) to Uint256 (hex)
  * Legacy support Export
+ * @param bn BigNumberish value to convert to Uint256 (hex)
+ * @returns {Uint256} Uint256 (hex) representation of the input BigNumberish value
+ * @example
+ * ```typescript
+ * const bnValue: BigNumberish = 28139218908760698114965631183142736658067655190345294133754686837148899816404n;
+ * const result = bnToUint256(bnValue);
+ * // result = "0x2fd23d9182193775423497fc0c472e156c57c69e4089a1967fb288a2d84e914"
```

Comment:
This claim is false and can't be falser 🗡️

Reply:
You have to take also into account that it has to be a code that a user can write and use.
bnToUint256() is not directly accessible for Starknet.js users.
To have to use : `uint256.bnToUint256()`

Reply:
Appologies for the false calm. Corrections have been implemented